holidays/countries/syrian_arab_republic.py

```diff
@@ -0,0 +1,141 @@
+from holidays.holiday_base import HolidayBase
+
+
+class SyrianArabRepublic(HolidayBase, InternationalHolidays, IslamicHolidays, StaticHolidays):
+    """
+    Syrian Arab Republic holidays.
+
+    References:
+    * [Timeanddate.com](https://www.timeanddate.com/holidays/syria/2025)
+    * [OfficeHolidays.com](https://www.officeholidays.com/countries/syria/2025)
+    * [PublicHolidays.me](https://publicholidays.me/syria/2025-dates/)
+    * [Wikipedia](https://en.wikipedia.org/wiki/Public_holidays_in_Syria)
+
+    Holidays are regulated by the Syrian government and may vary each year.
+    """
+
+    country = "SY"
+    default_language = "ar"
+    estimated_label = tr("(تقدير) %s")
+    supported_categories = (GOVERNMENT, OPTIONAL, PUBLIC)
+    supported_languages = ("ar", "en_US")
+    start_year = 1946  # Syria's independence
+
+    def __init__(self, islamic_show_estimated: bool = True, *args, **kwargs):
+        """
+        Args:
+            islamic_show_estimated:
+                Whether to add "estimated" label to Islamic holidays name
+                if holiday date is estimated.
+        """
+        InternationalHolidays.__init__(self)
+        IslamicHolidays.__init__(
+            self, cls=SyrianArabRepublicIslamicHolidays, show_estimated=islamic_show_estimated
+        )
+        StaticHolidays.__init__(self, SyrianArabRepublicStaticHolidays)
+        super().__init__(*args, **kwargs)
+
+    def _add_christmas_day(self):
+        """Add Christmas Day to the list of holidays."""
+        self._add_holiday("Christmas Day", 12, 25)
+
+    def _populate_public_holidays(self):
+        # New Year's Day
+        self._add_new_years_day(tr("رأس السنة الميلادية"))
+
+        # Revolution Day
+        self._add_holiday_mar_8(tr("عيد الثورة"))
+
+        # Mother's Day
+        self._add_holiday_mar_21(tr("عيد الأم"))
+
+        # Evacuation Day (Independence Day)
+        self._add_holiday_apr_17(tr("عيد الجلاء"))
+
+        # Labor Day
+        self._add_labor_day(tr("عيد العمال"))
+
+        # Martyrs' Day
+        self._add_holiday_may_6(tr("عيد الشهداء"))
+
+        # October Liberation Day
+        self._add_holiday_oct_6(tr("عيد حرب تشرين التحريرية"))
+
+        # Christmas Day
+        self._add_christmas_day()
+
+        # Islamic Holidays
+        self._add_eid_al_fitr_day(tr("عيد الفطر"))
+        self._add_eid_al_fitr_day_two(tr("عطلة عيد الفطر"))
+        self._add_eid_al_fitr_day_three(tr("عطلة عيد الفطر"))
+
+        self._add_arafah_day(tr("وقفة عرفة"))
+        self._add_eid_al_adha_day(tr("عيد الأضحى"))
+        self._add_eid_al_adha_day_two(tr("عطلة عيد الأضحى"))
+        self._add_eid_al_adha_day_three(tr("عطلة عيد الأضحى"))
+
+        self._add_islamic_new_year_day(tr("رأس السنة الهجرية"))
+        self._add_mawlid_day(tr("عيد المولد النبوي"))
+
+        # 27th of Rajab – Isra and Miraj
+        self._add_lunar_holiday(self._year, 7, 27, tr("الإسراء والمعراج"), "Isra and Miraj")
```

Comment:
I'm surprised you've came across all other Islamic holidays entries but not this
```suggestion
        # Isra' and Mi'raj.
        self._add_isra_and_miraj_day(tr("الإسراء والمعراج"))
```